app/clients/sms/twilio.py

```diff
@@ -226,6 +228,13 @@ def send_sms(
 
                     self.logger.info('Twilio sender has sms_sender_specifics')
 
+            # Log how long it spent in our system before we sent it
+            self.logger.info(
+                'Total time spent to send %s notification: %s seconds',
+                SMS_TYPE,
+                (datetime.utcnow() - created_at).total_seconds(),
+            )
+
```

Comment:
same question as above, since we haven't handed it off until the following lines, shouldn't the log come after we send it?